pyms/flask/services/opentelemetry.py

```diff
@@ -0,0 +1,129 @@
+import logging
+import time
+from typing import Text
+
+from flask import Blueprint, Response, request
+from pyms.flask.services.driver import DriverService
+
+from opentelemetry import metrics
+from opentelemetry.exporter.prometheus import PrometheusMetricsExporter
```

Comment:
Do you have an opinion about optional imports? With this service there will be a lot of imports, at least one per instrumentation. 
Should I be doing the same done on traces here? https://github.com/python-microservices/pyms/blob/master/pyms/flask/services/tracer.py#L5

Reply:
Why not import using https://github.com/pallets/werkzeug/blob/master/src/werkzeug/utils.py#L778 - it's already a flask dependency.